src/js/Content/Features/Store/Common/FAddToCartNoRedirect.js

```diff
@@ -35,7 +35,18 @@ export default class FAddToCartNoRedirect extends Feature {
                     return null;
             }
 
-            return document.forms[formName];
+            let formEl = document.forms[formName];
+
+            // Special handling for bundles on wishlist
+            if (onWishlist && !formEl && fnName === "addBundleToCart") {
+                formEl = document.forms[`add_to_cart_${href.match(/\d+/)[0]}`];
+                const inputEl = formEl?.elements.subid;
+                if (inputEl) {
+                    inputEl.name = "bundleid";
+                }
```

Comment:
I don't get the logic behind this, please add comments

Reply:
For bundles, the associated form name is wrong, and the `input` element with the name `subid` should be changed to `bundleid` otherwise the item will fail to add to cart. Steam has fallback logic in `addBundleToCart()` in case the form element is missing, but I figured we can just modify the broken form directly. But I'll add some comments.